app/page.tsx

```diff
@@ -1,27 +1,12 @@
-"use client";
+// LIBRARY IMPORTS
+import client from "@/tina/__generated__/client";
 
-//LIBRARY IMPORTS
-import { Link } from "@chakra-ui/next-js";
-import { Center, Box, Grid } from "@chakra-ui/react";
-import Image from "next/image";
+// LOCAL IMPORTS
+import Home from "./components/Home";
 
-//LOCAL IMPORTS
-import Hero from "./components/Hero";
-import CompanyValues from "./components/CompanyValues";
+export default async function Page() {
+  // CMS QUERIES
+  const result = await client.queries.page({ relativePath: "home.md" });
 
-export default function Home() {
-  const divStyle = {
-    minHeight: "1000px",
-    backgroundColor: "var(--primary-red)",
-    margin: "12px",
-    padding: "12px",
-    fontSize: "16px",
-  };
-
-  return (
-    <Grid templateColumns={"1fr"}>
-      <Hero />
-      <CompanyValues />
-    </Grid>
-  );
+  return <Home {...result} />;
```

Comment:
I had to refactor the code and create an extra outer 'layer' bc the data query is async, and that can't happen in client-side components. So I made this a server-side component, queried the data, and passed all the data as props. The Home component has now moved to the components folder.

Reply:
Sounds good!